package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@mapbox/geojson-rewind": "0.5.2",
-    "@xmldom/xmldom": "0.8.3",
+    "@xmldom/xmldom": "0.8.4",
```

Comment:
Latest version: https://www.npmjs.com/package/@xmldom/xmldom?activeTab=versions

Reply:
Is the exact version of @xmldom/xmldom important? Why not ^0.8.4, which would have allowed projects that depend on osmtogeojson to upgrade the version to a non-vulnerable one without changing this repo?

Reply:
Good point. There aren’t any reported vulnerabilities in 0.8.4 and 0.8.5 (at least now): https://security.snyk.io/package/npm/@xmldom%2Fxmldom

Using `^0.8.4` would do!

Reply:
@kachkaev Please note that I'm not a maintainer of this package. I'm just wondering, as I'm here for the same reason as you (critical vulnerability in our project), but no other dependency is noted with ^ in this package. They all use exact versions.

Reply:
Yep. That's a reason why I’ve flagged the existence of 0.8.6. If the versions have to be fixed, it’s best to use the latest available just in case.